src/components/AdminPage/AdminLayout.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import TopMenu from './TopMenu';
+import Articles from './Articles';
+import SideNavBar from './SideNavBar';
+
+function Layout() {
```

Comment:
Component name should be same as file name. `AdminLayout` in this case. I would rename the component and file to `AdminPage` though.